.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+
+# Issue
+
+<!--
+Mention if the PR fixes or address an issue in this section
+https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
+Example
+Fixes #999
+If this is a bug fix that does not have an issue created for it, please create one with enough info to reproduce the issue
+--->
+
+# Author(s)
```

Comment:
```suggestion
## Author(s)
```

Reply:
If this PR involves fixing a bug that does not already have an issue created for it, please create an issue with sufficient info to reproduce the problem.  Make sure to cross reference that issue within this PR.
--->